_eo_floorsets/041.md

```diff
@@ -36,4 +50,7 @@ boss_notes:
   - "Elements not important for Left/Rightbreathed. For other abilities, pay
   attention to the order of elements in the ability name - ice/cold is
   pointblank and thunder is donut"
+  - "All ice/thunder AoEs deal 900 potency magic damage (~50k to a DPS/healer)
+  and inflict stacking vulnerability up (20% per stack, 1m) along with
+  frostbite or paralysis (15s, uncurable with Esuna)"
```

Comment:
Ram's Voice causes deep freeze, not frostbite. I learned this today, thankfully with steel 😅 

Also thinking about rewriting these abilities slightly since the actual pointblank/donut AoEs are Ram's Voice and Dragon's Voice. Cold Thunder is actually just the charge AoE and is followed by Ram's > Dragon's, which are both instant (no cast). I'd guess that the Song abilities don't actually do anything except trigger the Voices - but I don't want to test it! Open to ideas, and I can make these changes later if you'd prefer

Reply:
Deep freeze fixed (yikes - but at least you survived!)

I was actually thinking about doing the same thing but couldn't make up my mind about the best solution, since you don't ever see Ram's/Dragon's Voice except if you look at the battle log. It's a fair point that the song abilities themselves don't deal damage (pretty sure of that from my run today), so I'll mark those as potency "n/a" and add in the Voice abilities with a "never used alone" note - at least then the ability list is accurate, and we can think about ways to clean it up from there.

Reply:
Yeah, I think that's better, thanks!